apps/web/src/routes/_content/resources/books/$bookId.tsx

```diff
@@ -31,14 +38,19 @@ function Book() {
     resourceId: +bookId,
   });
 
+  const { data: suggestedBooks, isFetched: isFetchedSuggestedBooks } =
+    trpc.content.getBooks.useQuery({
+      language: i18n.language,
+    });
+
   function displayAbstract() {
     return (
       book?.description && (
-        <div className="mt-6 border-l-4 border-blue-600 pl-4">
-          <h3 className="mb-4 text-lg font-semibold text-blue-900">
+        <div className="mt-5 lg:mt-8">
+          <h3 className="mb-4 lg:mb-5 body-12px-medium md:text-base font-medium text-white">
```

Comment:
use existing font classes (like subtitle-large-med-20px) + mobile font size shouldn't be this one